voter-dapp/README.md

```diff
@@ -1,6 +1,71 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Voter dApp
 
-## Available Scripts
+This is a browser client that UMA token holders can use to vote on price requests. The dApp interacts with the deployed [DVM](https://docs.umaproject.org/uma/oracle/technical_architecture.html) code using MetaMask's browser extension. The source code was bootstrapped from [Create React App](https://github.com/facebook/create-react-app).
+
+Features currently include:
+
+- Deploy a [designated voting proxy contract](https://docs.umaproject.org/uma/oracle/voting_with_UMA_2-key_contract.html)
+- Commit an encrypted vote
+- Backup salts associated with latest encrypted votes
+- Reveal an encrypted vote
+- Retrieve rewards for resolved votes
+- View results of resolved votes
+
+Note: this project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Moved this line into this section so it fit within the readme rather than awkwardly sitting at the top.